lib/controllers/situations.js

```diff
@@ -68,3 +68,74 @@ exports.openfiscaResponse = function(req, res, next) {
         res.send(result);
     });
 };
+
+function createTestContent(situation, test) {
+    var openfiscaRequest = mapping.buildOpenFiscaRequest(situation, true);
+    var testCase = openfiscaRequest.scenarios[0].test_case;
+    return _.assign(test, {
+        period: openfiscaRequest.scenarios[0].period,
+        absolute_error_margin: 10,
+        individus: testCase.individus,
+        familles: testCase.familles[0],
+        foyers_fiscaux: testCase.foyers_fiscaux[0],
+        menages: testCase.menages[0]
+    });
+}
+
+function myToString(obj) {
+    return obj.toString();
+}
+
+function normalizeTestContent(testFileContent) {
+    testFileContent.individus = testFileContent.individus.map(function(individu) {
+        individu.id = individu.id.toString();
+        return individu;
+    });
+
+    function replaceElementsWithStringRepresentation(obj) {
+        function convert(leafName) {
+            obj[leafName] = obj[leafName].map(myToString);
+        }
+        return convert;
+    }
+
+    var sources = {
+        familles: ['parents', 'enfants'],
+        foyers_fiscaux: ['declarants', 'personnes_a_charge'],
+        menages: ['enfants'],
+    };
+
+    for (var parent in sources) {
+        sources[parent].forEach(replaceElementsWithStringRepresentation(testFileContent[parent]));
+    }
+
+    testFileContent.menages.personne_de_reference = testFileContent.menages.personne_de_reference.toString();
+    if (testFileContent.menages.conjoint) {
```

Comment:
> This test could be removed, and the overall logic simplified, by making the iterator above test if it called on an `Array` and `map` accordingly, and call `toString` directly otherwise. This way, `personne_de_reference` and `conjoint` could be added directly as exported properties.